services/121-service/src/payments/fsp-integration/intersolve-visa/intersolve-visa.service.ts

```diff
@@ -545,7 +532,7 @@ export class IntersolveVisaService
   // TODO: Fix and this function as code it depends on has changed with the Visa re-implementation.
   // TODO: Re-implement and refactor this function according to new Module dependency model and encapsulate API details in IntersolveVisaApiService
   public async toggleBlockWallet(
-    tokenCode: string | null,
+    tokenCode: string,
```

Comment:
| null vanished? Should be there because of strict null check?

Reply:
It was not needed anymore since that function is only called when the variable is a string